### PR TITLE
Create search.md

### PR DESCRIPTION
**Pull request by Michellehbn**
@mishig25 do you mind taking a look and adding more info please, if needed? I was thinking to add this as a section under "Other" since it spans models, datasets, spaces - wdyt?

### CODE REVIEW

File: docs/hub/search.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Search
+
+You can now easily search anything on the Hub with **Full-text search**. We index all model cards, dataset cards, and Spaces app.py files.
```
Comment (julien-c) [inline docs/hub/search.md:3]:
```suggestion
You can now easily search anything on the Hub with **Full-text search**. We index model cards, dataset cards, and Spaces app.py files.
```

File: docs/hub/search.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Search
+
+You can now easily search anything on the Hub with **Full-text search**. We index model cards, dataset cards, and Spaces app.py files.
+
+Using the search bar at the top of https://huggingface.co, you can select "Try Full-text search" to find what you seek on the Hub across models, datasets, and Spaces:
+
+![full text search](https://github.com/huggingface/hub-docs/assets/83347449/2a66878d-dc2b-494f-a1f8-4ce12f764892)
+
+![albert](https://github.com/huggingface/hub-docs/assets/83347449/fa5921e8-ad0f-419b-899d-74cb805765f8)
```
Comment (osanseviero) [inline docs/hub/search.md:9]: The images need to be in https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub and in dark mode as well, see https://github.com/huggingface/hub-docs/blob/main/docs/hub/doi.md?plain=1#L32C1-L35 as an example for this

File: docs/hub/search.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Filter with ease: 
+
+![Filter search](https://github.com/huggingface/hub-docs/assets/83347449/209d40f1-da06-443d-9089-3f09b271d146)
+
+Available in both light and dark themes.
```
Comment (osanseviero) [inline docs/hub/search.md:15]: I would remove this

**Review by osanseviero — COMMENTED** [review]
Thanks!

**Comment by Michellehbn** [comment]
@mishig25 will be taking a look at this soon to update and add images! ❤️

**Comment by Michellehbn** [comment]
@mishig25 I've added the images! 😎

**Review by mishig25 — APPROVED** [review]
lgtm!

**Pull request merged**